FirebaseInAppMessaging/Sources/Private/Analytics/FIRIAMClientInfoFetcher.h

```diff
@@ -16,17 +16,18 @@
 
 #import <Foundation/Foundation.h>
 
-// A class for wrapping the interactions for retrieving client side info to be used in request
-// parameter for interacting with Firebase iam servers.
+// A class for wrapping the interactions for retrieving client side info to be used in
+// request parameter for interacting with Firebase iam servers.
 
 NS_ASSUME_NONNULL_BEGIN
 @interface FIRIAMClientInfoFetcher : NSObject
-// Fetch the up-to-date Firebase instance id and token data. Since it involves a server interaction,
+// Fetch the up-to-date FID and FIS token data. Since it involves a server interaction,
```

Comment:
I suspect that FID and FIS acronyms are not really common and may be misleading without additional context. I would suggest being a bit more verbose in comments, e.g. `Fetch the up-to-date Firebase Installations ID (FID) and token (FIStoken) data...`

Reply:
Improved docs in 53bce58.